pkgs/development/php-packages/uv/default.nix

```diff
@@ -0,0 +1,17 @@
+{ buildPecl, lib, libuv }:
+
+buildPecl {
+  pname = "uv";
+
+  version = "0.3.0";
+  sha256 = "sha256-oR+cmH9MRmtwx5BQcU/PDJFnU8VgQhJ5u+D2t9xzfT4=";
+
+  buildInputs = [ libuv ];
+
+  meta = with lib; {
```

Comment:
I missed to report the `platform` meta attribute in my first review.

Reply:
Alright, I added it.